subsys/partition_manager/Kconfig

```diff
@@ -6,6 +6,25 @@
 
 menu "Partition Manager"
 
+config PARTITION_MANAGER_ENABLED
+	bool "Partition manager is enabled"
+	help
+	  Option which can be checked to see whether or not the
+	  partition manager is enabled in the current build. Note that this
+	  should ideally be a hidden option, but due to limitations in how
+	  parent images impose options on child images it can't. Don't change
+	  the value of this option.
```

Comment:
```suggestion
	  should ideally be a hidden option, but it can't due to limitations
	  in how parent images impose options on child images. Don't
	  change the value of this option.
```
The sentence flows a bit better this way.